Flask-Integration/get-localhost.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, jsonify, request
+from spellCheck import *
+
+#export FLASK_APP=test1.py
+#flask run
+
+app = Flask(__name__)
+
+@app.route('/process')
+def dataGet():
```

Comment:
Can we update the name of this function to processSpellCheck or similar?

Reply:
Changed the name to processSpellCheck